src/index.ts

```diff
@@ -568,14 +573,16 @@ export async function assemble(config: Configuration, components: Components): P
     sharingDID: () => DID.sharing(components.crypto),
 
     // File system
-    loadFileSystem: (username: string) =>
-      loadFileSystem({ config, username, dependencies: components }),
-    recoverFileSystem: (params: RecoverFileSystemParams) =>
-      recoverFileSystem({
-        auth,
-        dependencies: { crypto: components.crypto, reference: components.reference, storage: components.storage },
-        ...params,
-      }),
+    loadFileSystem: (username: string) => loadFileSystem({ config, username, dependencies: components }),
+    recoverFileSystem: (params: RecoverFileSystemParams) => recoverFileSystem({ auth, dependencies: components, ...params }),
```

Comment:
Now that we have `fileSystem.load` and `fileSystem.recover`, should we remove the `loadFileSystem` and `recoverFileSystem` shorthands?

Reply:
We could yeah.